packages/nuxt3-module/plugin.ts

```diff
@@ -36,6 +39,16 @@ export default defineNuxtPlugin((NuxtApp) => {
         sameSite: "lax",
       });
     },
+    onErrorHandler(response) {
+      const error = isMaintenanceMode(response._data?.errors ?? []);
+      console.log("aaa", response._data?.errors);
```

Comment:
you mean "American Automobile Association" by this? :) please remove

Reply:
@mkucmus easier to add suggestion, we can commit without checking out the code :)
```suggestion
```